aws/internal/service/ebs/waiter/status.go

```diff
@@ -0,0 +1,40 @@
+package waiter
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	tfec2 "github.com/terraform-providers/terraform-provider-aws/aws/internal/service/ec2"
+)
+
+func importAwsEbsSnapshotRefreshFunc(conn *ec2.EC2, importTaskId string) resource.StateRefreshFunc {
```

Comment:
It looks like this should be named something like this one: https://github.com/grahamc/terraform-provider-aws/blob/0cf99fbd029ed7834576bc0d546e7738460f601e/aws/internal/service/datasync/waiter/status.go#L15

like `EbsSnapshotImportStatus`. I think the rest of this looks okay to me.